app/views/shared/sidebar/_projects.html.erb

```diff
@@ -1,27 +1,16 @@
 <ul>
-  <% if @projects_tree.present? then %>
+  <% if @projects_tree.present? %>
```

Comment:
Is this check even needed, since we have a loop afterwards?

Reply:
Probably not